packages/mikro-orm/README.md

```diff
@@ -0,0 +1,71 @@
+<p align="center">
+   <br/>
+   <a href="https://next-auth.js.org" target="_blank"><img height="64px" src="https://next-auth.js.org/img/logo/logo-sm.png" /></a>&nbsp;&nbsp;&nbsp;&nbsp;<img height="64px" src="https://raw.githubusercontent.com/nextauthjs/adapters/main/packages/mikro-orm/logo.svg" />
+   <h3 align="center"><b>Mikro ORM Adapter</b> - NextAuth.js</h3>
+   <p align="center">
+   Open Source. Full Stack. Own Your Data.
+   </p>
+   <p align="center" style="align: center;">
+      <img src="https://github.com/nextauthjs/adapters/actions/workflows/release.yml/badge.svg" alt="CI Test" />
+      <a href="https://www.npmjs.com/package/@next-auth/mikro-orm-adapter" target="_blank"><img src="https://img.shields.io/bundlephobia/minzip/@next-auth/mikro-orm-adapter/next" alt="Bundle Size"/></a>
+      <a href="https://www.npmjs.com/package/@next-auth/mikro-orm-adapter" target="_blank"><img src="https://img.shields.io/npm/v/@next-auth/mikro-orm-adapter/next" alt="@next-auth/mikro-orm-adapter Version" /></a>
+   </p>
+</p>
+
+## Overview
+
+This is the Mikro ORM Adapter for [`next-auth`](https://next-auth.js.org). This package can only be used in conjunction with the primary `next-auth` package. It is not a standalone package.
+
+You can find the Prisma schema in the docs at [next-auth.js.org/adapters/mikro-orm](https://next-auth.js.org/adapters/mikro-orm).
+
+## Getting Started
+
+1. Install `next-auth@beta` and `@next-auth/mikro-orm-adapter@next`
+
+   ```js
+   npm install next-auth@beta @next-auth/mikro-orm-adapter@next
+   ```
+
+2. Add this adapter to your `pages/api/[...nextauth].js` next-auth configuration object.
+
+   ```typescript
+   import NextAuth from "next-auth"
+   import { MikroOrmAdapter, Account, Session, User, VerificationToken } from "@next-auth/mikro-orm-adapter"
+
+   const config: Options<PostgreSqlDriver> = {
+      entities: [User, Session, Account, VerificationToken],
+      ...
+   };
+
+   // fetches a global instance of mikro-rom
+   const getORM = async () => {
+      if (!global.__MikroORM__) {
+         global.__MikroORM__ = await MikroORM.init({
+            dbName: "./db.sqlite",
+            type: "sqlite",
+            entities: [User, Session, Account, VerificationToken],
+            debug: process.env.DEBUG === "true" || process.env.DEBUG?.includes("db"),
+         })
+      }
+      return global.__MikroORM__
+   }
+
+
+   // For more information on each option (and a full list of options) go to
+   // https://next-auth.js.org/configuration/options
+   export default NextAuth(req, res, {
```

Comment:
```suggestion
   import { MikroOrmAdapter } from "@next-auth/mikro-orm-adapter"

   // fetches a global instance of mikro-rom
   const getORM = async () => {
      if (!global.__MikroORM__) {
         global.__MikroORM__ = await MikroORM.init({
            dbName: "./db.sqlite",
            type: "sqlite",
            entities: [User, Session, Account, VerificationToken],
            debug: process.env.DEBUG === "true" || process.env.DEBUG?.includes("db"),
         })
      }
      return global.__MikroORM__
   }


   // For more information on each option (and a full list of options) go to
   // https://next-auth.js.org/configuration/options
   export default NextAuth({
```

Reply:
Remember, we are trying to do this as simple as possible. It shouldn't be necessary by default to import anything else than the adapter itself.

What `mikro-orm` seems to be doing reminds me of this Prisma suggestion very much btw: https://www.prisma.io/docs/support/help-articles/nextjs-prisma-client-dev-practices#solution

Reply:
I think that kind of approach is pretty common in the nextjs world, especially with vercel, because the "global" scope is often "warmed" up on consecutive requests and thus your connection might already be initialized.

Reply:
Still seeing the extra imports from the adapter. Since we know the default entities, isn't there a way to just let the user not pass those at init?

I want the initialization to be as simple as this, or as close as we can get: 
![image](https://user-images.githubusercontent.com/18369201/143786210-37338ea0-b17c-4dfe-bc55-c62a7e834ef7.png)

Reply:
My suggestion commits haven't been pushed, and now there are conflicts... :sweat_smile: I'll try to fix them and push

Reply:
ah, damn.